core/domain/user_services_test.py

```diff
@@ -1353,6 +1353,17 @@ def test_get_human_readable_user_ids(self):
 
         self.assertEqual(user_ids, expected_user_ids)
 
+    def test_created_on_gets_updated_correctly(self):
+        # created_on should not be updated upon updating other attributes of
+        # the user settings model.
+        user_services.create_new_user('gae_id', 'user@example.com')
+        user_settings_model = user_models.UserSettingsModel.get_by_gae_id(
+            'gae_id')
+        time_of_creation = user_settings_model.created_on
+
+        user_services.update_user_bio('gae_id', 'New bio.')
+        self.assertEqual(user_settings_model.created_on, time_of_creation)
```

Comment:
Could you add a line above this: user_settings_model = user_models.UserSettingsModel.get_by_id(), so that we can make sure that this refers to the new user_settings_model and not the old one?

Also please do get_by_Id() instead of get_by_gae_id() -- this is a safeguard to make sure the id still works when you set it using the "populate" method.

Reply:
Got it. Done, thanks!